index.rst

```diff
@@ -268,8 +269,8 @@ Environmental
     MS5611, components/sensor/ms5611, ms5611.jpg, Pressure
     NTC Thermistor, components/sensor/ntc, ntc.jpg, Temperature
     RuuviTag, components/sensor/ruuvitag, ruuvitag.jpg, Temperature & Humidity & Accelerometer
-    SCD30, components/sensor/scd30, scd30.jpg, Temperature & Humidity
     SDP3x, components/sensor/sdp3x, sdp31.jpg, Pressure
+    SCD30, components/sensor/scd30, scd30.jpg, CO2 & Temperature & Humidity
```

Comment:
Can we move it up again to retain the alphabetical sorting?

Reply:
Done!